solutions/remotemonitoring/scripts/individual/deployment-configmap.yaml

```diff
@@ -76,20 +50,10 @@ data:
   security.cors.whitelist: ""
   security.application.secret: "{Application secret}"
 
-  # Azure maps key
-  azure.maps.key: "{AzureMapsKey}"
-
-  # ASA Manager
-  asa.eventhub.connstring: "{Event hub connection string}"
-  asa.eventhub.name: "{Event hub name}"
-
   # Diagnostics
   diagnostics.webservice.url: "http://diagnostics-svc:9006/v1"
-  diagnostics.cloud.type: "{Cloud environment name}"
   diagnostics.subscription.id: "{Subscription id}"
   diagnostics.iothub.name: "{Iot Hub name}"
-  diagnostics.deployment.id: "{Deployment Id of the solution}"
-  diagnostics.appinsights.instrumentationkey: "{Diagnostics Appinsights instrumentaion key}"
   diagnostics.solution.type: "{Solution type}"
   diagnostics.solution.name: "{Solution name}"
```

Comment:
Why did we leave some diagnostics values in?

Reply:
This was before our plan to use Sushil's script. This is updated further in the 3rd PR.